dev-resources/puppetlabs/services/legacy_routes/legacy_routes_test/confdir/puppet.conf

```diff
@@ -1,4 +1,4 @@
-[master]
+[server]
```

Comment:
I think this whole file can go, we ripped out the legacy routes test in this branch, but forgot this.

Reply:
I'll put up something separate for that.